packages/repeater/src/lib/Repeater.spec.ts

```diff
@@ -127,4 +142,104 @@ describe('Repeater', () => {
       ).once();
     });
   });
+
+  describe('runningStatus', () => {
+    it('should have RunningStatus.OFF initially', () => {
+      expect(repeater.runningStatus).toBe(RunningStatus.OFF);
+    });
+
+    it('should have RunningStatus.STARTING just after start() call', () => {
+      void repeater.start();
+      expect(repeater.runningStatus).toBe(RunningStatus.STARTING);
+    });
+
+    it('should have RunningStatus.RUNNING after successful start()', async () => {
+      await repeater.start();
+      expect(repeater.runningStatus).toBe(RunningStatus.RUNNING);
+    });
+
+    it('should have RunningStatus.OFF after start() and stop()', async () => {
+      await repeater.start();
+      await repeater.stop();
+      expect(repeater.runningStatus).toBe(RunningStatus.OFF);
+    });
+
+    it('should throw an error on start() twice', async () => {
+      await repeater.start();
+
+      const res = repeater.start();
+
+      await expect(res).rejects.toThrow('Repeater is already active.');
+    });
+
+    it('should be possible to start() after start() error', async () => {
+      when(mockedEventBus.execute(anyOfClass(RegisterRepeaterCommand)))
+        .thenReject()
+        .thenResolve({ version });
+
+      await expect(repeater.start()).rejects.toThrow();
+      await expect(repeater.start()).resolves.not.toThrow();
+    });
+
+    it('should throw an error on stop() without start()', async () => {
+      const res = repeater.stop();
+
+      await expect(res).rejects.toThrow('Cannot stop non-running repeater.');
+    });
+
+    it('should throw an error on stop() twice', async () => {
+      await repeater.start();
+      await repeater.stop();
+
+      const res = repeater.stop();
+
+      await expect(res).rejects.toThrow('Cannot stop non-running repeater.');
+    });
+  });
+
+  describe('process termination', () => {
```

Comment:
However, we have to revise the test titles as well.

```suggestion
  describe('constructor', () => {
```